src/dom.ts

```diff
@@ -20,6 +20,50 @@ for (const param in tagWrap) {
 	tw.post = '</' + tw.reverse().join('></') + '>';
 }
 
+function validateClass(token: string): void {
+	if (!token || token === '') {
+		throw new Error('An invalid or illegal string was specified');
+	}
+	if (/\s/.test(token)) {
+		throw new Error('String contains an invalid character');
+	}
+}
+
+/**
+ * Adds one or more CSS class names to an Element without duplication
+ *
+ * @param element The Element to which to add CSS classes
+ * @param classes An array of string CSS classes to add to the Element
+ *
+ * @example
+ * dom.addClass(document.body, 'loaded');
+ *
+ * @example
+ * dom.addClass(document.body, 'loaded', 'ready');
+ */
+export function addClass(element: Element, ...classes: string[]): void {
+	let targetElement = <any> element;
+	if (!targetElement) {
+		return;
+	}
+	if (targetElement.classList) {
+		let classList: any = targetElement.classList;
+		classList.add.apply(classList, classes);
```

Comment:
This is overly naive.  Not all browsers that support `classList` support multi-argument `add` and `remove` (this would fail in IE11, for example, only adding the first class).

We have a couple of options:

* I proposed has features for multi-argument add/remove and 2-argument toggle in the polyfill section of the dom proposal (obviously we're not doing polyfills right now so it would've been easy to overlook that).  It seems like it should be reasonable to add generic has tests for multi-argument support in these methods when they exist.  We would still need to test `targetElement.classList` directly on each call since IE doesn't support `classList` in SVG elements.
* The other option is to just go with the `else` branch below in all cases.